kakaemma/social-networks
# #179318685 add testing to the social networks

#### What does this PR do?
This PR allows for tests with Rspec  and code coverage to the application
#### Description of Task to be completed?
- add Rspec
- add code coverage with simplecov
- add robocop to the application

#### How should this be manually tested?
- pull and checkout to this branch
- run `bundle install`
- run `bundle exec rspec`
-  You should be able to see  `zero tests run and zero failure`. a coverage folder should also be generated

#### What are the relevant pivotal tracker stories?
[#179318685](https://www.pivotaltracker.com/story/show/179318685)

#### Implementation Plan link(s)
[Social Networks | add tests and coverage Implementation Plan](https://docs.google.com/document/d/15Gfi5bMlG1-vzBlQfnQSuvmCsksnq7cpaUR4kI_nYGk/edit?usp=sharing)
#### Any background context you want to add?
N/A
